src/netaddress.cpp

```diff
@@ -204,18 +204,22 @@ enum Network CNetAddr::GetNetwork() const
     return NET_IPV6;
 }
 
-std::string CNetAddr::ToStringIP() const
+std::string CNetAddr::ToStringIP(bool name_resolve) const
```

Comment:
Why is this being changed with this pull request?

Reply:
Good catch. This stems from an earlier implementation of `xversion` (as an extension of the version message) where this behavior of ToStringIP(..) was a problem. I'll take it out.